package.json

```diff
@@ -30,8 +30,10 @@
   "homepage": "https://github.com/inclusive-design/standards.idrc.ocadu.ca#readme",
   "dependencies": {
     "@11ty/eleventy": "^3.0.0-beta.1",
```

Comment:
Can you merge changes from `main` into your branch? It's using v3 now.

Reply:
I am having some weird issue with Git, where I can't merge main branch to this one. CLI is keep saying that it is already up to date, but I don't see this package changes. Just letting you know that I am going to merge main for the dependency updates.

Reply:
I think you may need to add the upstream if you have cloned your own fork. Do you want to walk through that process?

Reply:
@chosww Can you push the merge we did yesterday?

Reply:
Hey @greatislander, happen to be that eleventy 3.0 update was only made to package-lock.json, and package.json is not updated in main branch. I thought it was an issue with git, but I already have the latest changes in this PR

Reply:
I've updated main for clarity (Eleventy 3.0.0 matched that version requirement but I think it's better to be clear that we aren't using a beta). Can you merge upstream main in?